e2e/atlas/helper.go

```diff
@@ -0,0 +1,117 @@
+package atlas
```

Comment:
I like the idea of this file, one general comment I would make `cliPath` a global var of the module so you don't have to pass it to each method

Reply:
another general thing here let's make this package `atlas_test`
so we can keep the methods private since I don't expect us to use them outside these tests

Reply:
It seems that in order to change the package name I have to move the helper inside the folder `atlas_test`.  Also after doing that and making the methods private I am getting a lint error which says that my methods are not being used. Should be private method visibility within the package? @gssbzn

Reply:
If you see the other tests in this folder are in `atlas_test` package, you only need to change the name of this file to be `helper_test` as well